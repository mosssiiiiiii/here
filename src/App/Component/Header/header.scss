@import "../../../setup/style/colors";

.header{
  background-color: $white;
  padding: 15px 0;
  img{
    width: 70px;
    padding-left: 15px;
  }
}
