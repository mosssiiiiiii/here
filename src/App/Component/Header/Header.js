import React from 'react';
import './header.scss';

function Header() {
    return (
        <div className='header'>
            <div className='limited'>
                <img src="/asset/logo.svg" alt=""/>
            </div>
        </div>
    )
}


export default Header;
