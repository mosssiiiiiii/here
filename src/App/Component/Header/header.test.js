import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';


it('renders correctly header component', () => {
    const header = renderer
        .create(<Header/>)
        .toJSON();
    expect(header).toMatchSnapshot();
});
