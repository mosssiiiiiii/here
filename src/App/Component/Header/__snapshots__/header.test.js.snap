// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly header component 1`] = `
<div
  className="header"
>
  <div
    className="limited"
  >
    <img
      alt=""
      src="/asset/logo.svg"
    />
  </div>
</div>
`;
